### DISCUSSION TITLE
Planetscale + Prisma ( same db / integration on vercel) has worked for 2 apps, but now its not working on new application.

### DISCUSSION THREAD

**Discussion by jxnl** [vercel/community#1700] [Help]
### Summary

I've had this one database w/ planetscale work on both flask and next apps all through the planetscale integration in vercel.

This time my app is just throwing me 

```
Can't reach database server at `us-east.connect.psdb.cloud`:`3306`
Please make sure your database server is running at `us-east.connect.psdb.cloud`:`3306`.
```

Working just fine locally with `npx prisma studio` using the same `DATABASE_URL` but in prod it can't be reached.

**Comment by JohnVersus** [comment]
Same here. Works on localhost with but not working on vercel after deployment. 
I am using dev dependency `"prisma": "4.11.0"` and  dependency `"@prisma/client": "4.11.0",`

**Reply by JohnMwanyika** [reply]
try adding prisma in your dependancy list not dev dependancy.

**Reply by JohnVersus** [reply]
But sometimes it works. It randomly throws an error. 
https://github.com/orgs/vercel/discussions/1700#discussioncomment-5376988

**Comment by amyegan** [comment]
Do you have an example deployment or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that you can share?

**Reply by JohnMwanyika** [reply]
Hello,,I'm using remote db hosted somewhere else on my cPanel now I need your Ip address so I can allow it to access my database from the remote mysql in cPanel. Have checked on vercel documentation but culdnt find any.. kindly Help,, I think this is the issue my colleagues are facing also

**Comment by JohnVersus** [comment]
Hi @amyegan 
I dont see a way to reproduce it. As per my vercel logs, these errors are totally random. Here is the complete error stack if it helps in any way
```
{ origin: undefined }
Unhandled Promise Rejection 	
{"errorType":"Runtime.UnhandledPromiseRejection","errorMessage":"Error: Timed out fetching a new connection from the connection pool. 
More info: http://pris.ly/d/connection-pool (Current connection pool timeout: 10, connection limit: 5)","reason":{"errorType":"Error","errorMessage":"Timed out fetching a new connection from the connection pool. More info: http://pris.ly/d/connection-pool (Current connection pool timeout: 10, connection limit: 5)","clientVersion":"4.11.0","errorCode":"P2024",
"stack":["Error: Timed out fetching a new connection from the connection pool. More info: http://pris.ly/d/connection-pool (Current connection pool timeout: 10, connection limit: 5)","    at startFn (/var/task/node_modules/@prisma/client/runtime/library.js:101:2598)","    at async updateAPIStats (/var/task/.next/server/chunks/899.js:24:5)"]},"promise":{},
"stack":["Runtime.UnhandledPromiseRejection: Error: Timed out fetching a new connection from the connection pool. More info: http://pris.ly/d/connection-pool (Current connection pool timeout: 10, connection limit: 5)","    
at process.<anonymous> (file:///var/runtime/index.mjs:1188:17)","    at process.emit (node:events:525:35)","   
at emit (node:internal/process/promises:149:20)","    at processPromiseRejections (node:internal/process/promises:283:27)","    at process.processTicksAndRejections (node:internal/process/task_queues:96:32)"]}
Unknown application error occurred
Runtime.Unknown

```
